src/commcare_cloud/fab/deploy_diff.py

```diff
@@ -32,6 +32,12 @@ def url(self):
         """Human-readable diff URL"""
         return "{}/compare/{}...{}".format(self.repo.html_url, self.last_commit, self.deploy_commit)
 
+    @property
+    def deployed_commit_matches_latest_commit(self):
+        if self.last_commit and self.deploy_commit:
+            short, long = sorted([self.last_commit, self.deploy_commit], key=lambda x: len(x))
+            return self.last_commit == self.deploy_commit or long.startswith(short)
```

Comment:
Is it intentional that this function implicitly returns `None` when the `if` condition fails?
```suggestion
            return self.last_commit == self.deploy_commit or long.startswith(short)
        return False
```